x-pack/legacy/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -56,6 +57,18 @@ export class HeadlessChromiumDriverFactory {
 
   type = BROWSER_TYPE;
 
+  async screenshotPage() {
+    if (!this.page) {
+      throw new Error(`Page creation hasn't happened yet`);
+    }
+
+    return this.page.screenshot({
+      type: 'jpeg',
```

Comment:
would `png` work here? I'm not sure we need to add a 3rd file type

Reply:
I did jpeg because you can specify a smaller file size, whereas `png` can be potentially big (since it's base64), plus slower to render etc